.eslintrc

```diff
@@ -30,6 +30,10 @@
       }
     ],
 
+    // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md#disallow-variable-declarations-from-shadowing-variables-declared-in-the-outer-scope-no-shadow
+    "no-shadow": "off",
```

Comment:
Where is this affected? I couldn't find where this rule is being an issue. From reading the documentation, I think it's useful to keep on still, but it's hard to say because I can't find the affected code.

Reply:
I'll have to see if I can reproduce it. I was getting errors from types defined in other files without this. My understanding is that the eslint check doesn't understand typescript and reports false errors, and that the next line replaces it with a TS_aware version.

Reply:
It's possible that the code has changed since this was an issue, though, in which case I can remove it.